Baikal/PostEffects/ML/thread_safe_queue.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+namespace ML {
+    ///< An implementation of a concurrent queue
```

Comment:
Just `///`? AFAIK, `///<` is for putting documentation after members.
But I guess we may use the original unmodified queue class.